packages/block-library/src/paragraph/block.json

```diff
@@ -7,7 +7,6 @@
 	"description": "Start with the basic building block of all narrative.",
 	"keywords": [ "text" ],
 	"textdomain": "default",
-	"usesContext": [ "postId" ],
```

Comment:
I've removed it per [this comment](https://github.com/WordPress/gutenberg/pull/53247#discussion_r1290501823) We can get the id from `$post->ID`.